Azure CXP Doc Edit - Title Update

Givary opened the pull request:
Updated title with the below information in line 2
'Tutorial: Configure Workday for automatic user provisioning with Active Directory
Previous title - 'Tutorial: Configure Workday for automatic user provisioning with Azure Active Directory
MicrosoftDocs/azure-docs/issues/87809

Givary committed:
Title Update

Updated title with the below information in line 2
'Tutorial: Configure Workday for automatic user provisioning with Active Directory
Previous title - 'Tutorial: Configure Workday for automatic user provisioning with Azure Active Directory

cmmdesai reviewed (changes requested):
Please use the word "on-premises" so it's clear.  
"Configure Workday for automatic user provisioning with **on-premises** Active Directory".

Givary committed:
Updated title

Updated title line # 2 - Configure Workday for automatic user provisioning with on-premises Active Directory
